spread-tests/hello-world-is-confined/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that hello-world.evil is confined
```

Comment:
What's actually being tested here?  We're of course not very interested in hello world itself.